src/components/ResultsItem/HoverableDiv.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import HoverVideo from './HoverVideo'
+
+const HoverableDiv = ({isVideo, isHovering, link,linkName,setIsHovering}) => {
+
+
+    return (
+        <div id="parent">
```

Comment:
could we set this div to have an `id` that is a bit more descriptive? I think "parent" is a bit too vague